packages/fx-core/resource/package.nls.json

```diff
@@ -501,8 +501,8 @@
   "core.selectValidateMethodQuestion.validate.schemaOptionDescription": "Validate using manifest schema",
   "core.selectValidateMethodQuestion.validate.appPackageOption": "Validate app package using validation rules",
   "core.selectValidateMethodQuestion.validate.appPackageOptionDescription": "Validate app package using validation rules",
-  "core.selectValidateMethodQuestion.validate.testCasesOption": "Publish Readiness",
-  "core.selectValidateMethodQuestion.validate.testCasesOptionDescription": "Check your app with Microsoft's test cases before publishing",
+  "core.selectValidateMethodQuestion.validate.testCasesOption": "Validate against pre-publish test cases",
+  "core.selectValidateMethodQuestion.validate.testCasesOptionDescription": "Comprehensive tests to assess readiness",
```

Comment:
Proposed: Comprehensive tests to ensure readiness

Reply:
Great suggestion, updated.